src/pages/index.js

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
 import {shape} from 'prop-types';
+import {graphql} from 'gatsby';
 import {CurrentMeeting} from '@dsmjs/components';
+import Layout from '../components/layout';
 
 export default function SiteIndex({data}) {
   const {frontmatter} = data.allMarkdownRemark.edges[0].node;
 
+  // TODO: this is janky, the @dsmjs/components CurrentMeeting component needs updated
+  const talk = {
+    ...frontmatter.talk.frontmatter,
```

Comment:
along with cleaning up the jankiness, id love for the component to not know about the `frontmatter` piece. is it possible to transform that out of what gets passed down to the component?

Reply:
to clarify, would love to not need `frontmatter` in cases like this: https://github.com/dsmjs/components/blob/2c2ec5d114a791ed83cef004a5bc4c79c9b66ac7/src/atoms/archive-list-item/index.js#L11

i imagine this may be in line with what you are already targetting?